app/controllers/idv/phone_controller.rb

```diff
@@ -4,23 +4,23 @@ class PhoneController < ApplicationController
     include IdvFailureConcern
 
     before_action :confirm_step_needed
-    before_action :confirm_step_allowed
+    before_action :confirm_step_allowed, except: [:failure]
     before_action :refresh_if_not_ready, only: [:show]
 
     def new
-      @view_model = view_model
+      @idv_form = idv_form
       analytics.track_event(Analytics::IDV_PHONE_RECORD_VISIT)
     end
 
     def create
-      result = idv_form.submit(step_params)
+      @idv_form = idv_form
```

Comment:
Why the use of `@idv_form` and `@_idv_form` (line 76)? I'd just use `idv_form` everywhere.